src/entities/Entity.js

```diff
@@ -0,0 +1,10 @@
+class Entity {
+    _id;
```

Comment:
This seems to be readonly, don't you need a way to set this at least for first time?

Reply:
It would be a gold idea to comment the data type of eche property

Reply:
Don't the IDs get created by the Database?  I don't need to suer to define the ID for the object, I just need the ID for updates/ deletes, not for insert, right? 

About "Commenting data type" : Sounds good. I've added the comments to properties in constructor and explicitly declared in class.

I'll provide a second comment once the new PR is ready.